File: app/src/main/java/com/lumiyaviewer/lumiya/ui/common/OnInterceptTouchEventListener.java
package com.lumiyaviewer.lumiya.ui.common;
import java.util.*;

import android.view.MotionEvent;

public interface OnInterceptTouchEventListener {
    boolean dispatchTouchEvent(MotionEvent motionEvent);

    boolean onInterceptTouchEvent(MotionEvent motionEvent);

    boolean onTouchEvent(MotionEvent motionEvent);
}

File: Linkpoint/src/main/app_broken_backup/com/linkpoint/ui/common/OnInterceptTouchEventListener.kt
package com.linkpoint.ui.common

import android.view.MotionEvent

interface OnInterceptTouchEventListener {
    fun dispatchTouchEvent(event: MotionEvent): Boolean

    fun onInterceptTouchEvent(event: MotionEvent): Boolean

    fun onTouchEvent(event: MotionEvent): Boolean
}
